### PR TITLE
[Fix] #166 #165

### PR DESCRIPTION
**Pull request by SujithThirumalaisamy**
Added the arrow search functionality and also fixed the ui bug in the landing page.

### CODE REVIEW

File: packages/ui/src/SearchDialog.tsx [hunk 1]
```diff
@@ -67,9 +127,9 @@ export function SearchDialog({ tracks }: { tracks: (Track & { problems: Problem[
             <span className="sr-only">Close</span>
           </DialogClose>
         </div>
-        <div className="h-[400px] overflow-y-scroll">
-          {searchTracks.map((track) => (
-            <TrackList key={track.id} track={track} />
+        <div className="h-[400px] overflow-y-scroll" onKeyDown={autoScroll} ref={searchFilterListRef}>
```
Comment (hkirat) [inline packages/ui/src/SearchDialog.tsx:130]: why the onKeydown here?

File: packages/ui/src/SearchDialog.tsx [hunk 2]
```diff
@@ -28,6 +32,61 @@ export function SearchDialog({ tracks }: { tracks: (Track & { problems: Problem[
...
+    };
+
+    window.addEventListener("keydown", handleKeyPress);
+
+    return () => {
+      window.removeEventListener("keydown", handleKeyPress);
+    };
+  }, [searchTracks, selectedTrack]);
+
+  const autoScroll = (e) => {
```
Comment (hkirat) [inline packages/ui/src/SearchDialog.tsx:86]: is this needed?

File: packages/ui/src/SearchDialog.tsx [hunk 3]
```diff
@@ -28,6 +32,61 @@ export function SearchDialog({ tracks }: { tracks: (Track & { problems: Problem[
...
+        router.push(`/tracks/${searchTracks[selectedTrack].id}/${searchTracks[selectedTrack].problems[0]?.id}`);
+      }
+    };
+
+    window.addEventListener("keydown", handleKeyPress);
+
+    return () => {
+      window.removeEventListener("keydown", handleKeyPress);
+    };
+  }, [searchTracks, selectedTrack]);
```
Comment (hkirat) [inline packages/ui/src/SearchDialog.tsx:84]: This'll reload everytime `selectedTrack` changes. Is there a better way of doing it without removing and re-attaching the handler?
Not a priority

File: README.md [hunk 4]
```diff
@@ -1,92 +1,24 @@
...
-install yarn in your system.
-
-install dependencies
-yarn install
-
-start development
-yarn run dev
-
-# Turborepo starter
-
```
Comment (hkirat) [inline README.md:12]: Can u split this into a dif PR?

**Review by hkirat — CHANGES_REQUESTED** [review]

File: packages/ui/src/SearchDialog.tsx (hunk 1 above)
Comment (SujithThirumalaisamy) [reply packages/ui/src/SearchDialog.tsx:130]: Added this to check if the next Tracklist is visible in the container. If not will make it visible.

Comment (SujithThirumalaisamy) [reply packages/ui/src/SearchDialog.tsx:130]: > Added this to check if the next Tracklist is visible in the container. If not will make it visible.

Sorry. Haven't cleared the testing things.

File: packages/ui/src/SearchDialog.tsx (hunk 2 above)
Comment (SujithThirumalaisamy) [reply packages/ui/src/SearchDialog.tsx:86]: Ahhhh not really

File: packages/ui/src/SearchDialog.tsx (hunk 3 above)
Comment (SujithThirumalaisamy) [reply packages/ui/src/SearchDialog.tsx:84]: figure this

**Pull request closed without merging**